src/pages/home/uploadSong/AdvancedSongDetails.tsx

```diff
@@ -0,0 +1,149 @@
+import AutoAwesomeIcon from "@mui/icons-material/AutoAwesome";
+import { Box, IconButton, Stack } from "@mui/material";
+import { useWindowDimensions } from "common";
+import {
+  DropdownMultiSelectField,
+  SwitchInputField,
+  TextInputField,
+} from "components";
+import { Button, DatePickerInput, HorizontalLine, Typography } from "elements";
+import { useFormikContext } from "formik";
+import { UploadSongRequest } from "modules/song";
+import theme from "theme";
+
+const AdvancedSongDetails: React.FC = () => {
+  const windowWidth = useWindowDimensions()?.width;
+
+  const { isSubmitting } = useFormikContext<UploadSongRequest>();
+
+  return (
+    <Stack>
```

Comment:
is this outter `Stack` needed? i think it can be removed as the only child is also another `Stack`